packages/react-forms/src/InputDate/index.js

```diff
@@ -36,126 +36,134 @@ type State = {
   isOpen: boolean,
 };
```

Comment:
I don't think we use the `State` type anymore

Reply:
Deleted.